PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -194,4 +208,62 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
 };
 } // namespace o2::hf_evsel
 
+namespace o2::hf_evsel_mc
+{
+// MC collision rejection types
+enum McCollisionRejection {
+  None = 0,
+  Trigger,
+  TvxTrigger,
+  TimeFrameBorderCut,
+  ItsRoFrameBorderCut,
+  PositionZ,
+  NMcCollisionRejection
+};
+
+struct HfEventSelectionMc {
+  // event selection parameters (in chronological order of application)
+  bool useSel8Trigger{false};       // Apply the Sel8 selection
+  bool useTvxTrigger{false};        // Apply the TVX trigger
+  bool useTimeFrameBorderCut{true}; // Apply TF border cut
+  bool useItsRofBorderCut{false};   // Apply the ITS RO frame border cut
+  float zPvPosMin{-1000.f};         // Minimum PV posZ (cm)
+  float zPvPosMax{1000.f};          // Maximum PV posZ (cm)
+
+  /// \brief Function to apply event selections to generated MC collisions
+  /// \param mcCollision MC collision to test against the selection criteria
+  /// \return a bitmask with the event selections not satisfied by the analysed collision
+  template <typename TBc, typename TMcColl>
+  uint8_t getHfMcCollisionRejectionMask(TMcColl const& mcCollision)
+  {
+    uint16_t rejectionMask{0};
```

Comment:
Maybe better to keep it consistent with the non-MC mask.
```suggestion
  uint16_t getHfMcCollisionRejectionMask(TMcColl const& mcCollision)
  {
    uint16_t rejectionMask{0};
```